config/routes.rb

```diff
@@ -15,6 +15,8 @@
     resources :categories, only: [:index]
   end
 
+  resources :genres, only: [:new, :create, :index, :show]
```

Comment:
Since the new functionality lives on index, you don't need the route here or the action in the controller.